beets/ui/__init__.py

```diff
@@ -99,6 +99,12 @@ def print_(*strings):
     replaces it.
     """
     if strings:
+        if not isinstance(strings[0], basestring):
+            try:
+                strings = map(unicode, strings)
+            except UnicodeError:
+                strings = map(bytes, strings)
```

Comment:
We could also just require all the arguments to `print_` be unicode (or unicode-convertible). We could either just raise an error when bytes are passed or do a best-effort conversion.

The only problem with getting rid of `print_` is that, on some systems, the locale settings are so broken that not all unicode strings can be printed. For example, if you set Python up with an ASCII locale (e.g., LC_ALL=C), just calling `print(u'caf\xe9')` will crash. It's incredibly annoying, and hard to reproduce on systems like yours where the locale is reasonable. So, sadly, I think we need to keep the layer of indirection unless there's an obvious alternative that doesn't involve polluting global interpreter state.